addons/medical_treatment/functions/fnc_canStitch.sqf

```diff
@@ -18,4 +18,7 @@
 
 params ["", "_patient"];
 
-!(GET_BANDAGED_WOUNDS(_patient) isEqualTo [])
+!(GET_BANDAGED_WOUNDS(_patient) isEqualTo []) && {
```

Comment:
Don't need this, the `allowSelfTreatment` config entry handles this. [reference](https://github.com/acemod/ACE3/blob/master/addons/medical_treatment/functions/fnc_canTreat.sqf#L27)

Reply:
Didn't know canTreat worked like that, niiice.